app/src/sharedTest/java/org/simple/clinic/TestData.kt

```diff
@@ -1320,4 +1321,67 @@ object TestData {
       bloodSugars = bloodSugars,
       prescribedDrugs = prescribedDrugs
   )
+
+
+  fun contactPatientProfile(
+      patientUuid: UUID = UUID.randomUUID(),
+      patientAddressUuid: UUID = UUID.randomUUID(),
+      syncStatus: SyncStatus = randomOfEnum(SyncStatus::class),
+      generatePhoneNumber: Boolean = faker.bool.bool(),
+      generateBusinessId: Boolean = faker.bool.bool(),
+      patientStatus: PatientStatus = PatientStatus.Active,
+      patientDeletedAt: Instant? = null,
+      patientName: String = faker.name.name(),
+      patientPhoneNumber: String? = if (generatePhoneNumber) faker.phoneNumber.phoneNumber() else null,
+      businessId: BusinessId? = if (generateBusinessId) businessId(patientUuid = patientUuid) else null,
+      generateDateOfBirth: Boolean = faker.bool.bool(),
+      dateOfBirth: LocalDate? = if (generateDateOfBirth) LocalDate.parse("1980-01-01") else null,
+      age: Age? = if (!generateDateOfBirth) Age(value = kotlin.random.Random.nextInt(30..100), updatedAt = Instant.parse("2018-01-01T00:00:00Z")) else null,
```

Comment:
We already imported `nextInt`, so we can directly use that without `Random.nextInt`

Reply:
It keeps the package by default there not sure why. can pair with you whenever possible on this.

Reply:
Have you tried directly using `nextInt`? 🤔